lib/loaders/babel.js

```diff
@@ -12,11 +12,11 @@
 const loaderFeatures = require('../features');
 const applyOptionsCallback = require('../utils/apply-options-callback');
 
-/**
- * @param {WebpackConfig} webpackConfig
- * @return {Array} of loaders to use for Babel
- */
 module.exports = {
+    /**
```

Comment:
I'm reverting this to prevent conflicts with https://github.com/symfony/webpack-encore/pull/550/files#diff-35a173566334465c73f4aebaff4cc9eb